packages/react-components/react-drawer/src/components/DrawerFooter/DrawerFooter.types.ts

```diff
@@ -0,0 +1,15 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+
+export type DrawerFooterSlots = {
+  root: Slot<'footer'>;
+};
+
+/**
+ * DrawerFooter Props
+ */
+export type DrawerFooterProps = ComponentProps<DrawerFooterSlots> & {};
```

Comment:
Will there be more props here? Because if not maybe we can remote `& {}`

Reply:
Fixed.